src/library/library.cpp

```diff
@@ -369,6 +455,94 @@ QStringList Library::getDirs() {
     return m_pTrackCollection->getDirectoryDAO().getDirs();
 }
 
+void Library::paneCollapsed(int paneId) {
```

Comment:
I don't know an easy way to do this and there's much code related to the multi-pane behavior.